inngest-core/src/main/kotlin/com/inngest/Environment.kt

```diff
@@ -0,0 +1,13 @@
+package com.inngest
+
+object Environment {
+    fun inngestHeaders(framework: SupportedFrameworkName? = null): RequestHeaders {
+        val sdk = "inngest-kt:${Version.getVersion()}"
+        return mapOf(
+            InngestHeaderKey.ContentType.value to "application/json",
+            InngestHeaderKey.Sdk.value to sdk,
+            InngestHeaderKey.UserAgent.value to sdk,
```

Comment:
user agent isn't required by the SDK spec right? although it makes sense to set it the sdk string

Reply:
Right both the Python and TS SDK are adding it.